test/addons-napi/addons-napi.status

```diff
@@ -1,11 +0,0 @@
-prefix addons-napi
-
-# To mark a test as flaky, list the test name in the appropriate section
-# below, without ".js", followed by ": PASS,FLAKY". Example:
-# sample-test                        : PASS,FLAKY
-
-[true] # This section applies to all platforms
-
-[$system==win32]
-# https://github.com/nodejs/node/issues/23621
-test_threadsafe_function/test: PASS,FLAKY
```

Comment:
Given https://github.com/nodejs/node/issues/23621 is still open should this status file be moved/renamed to `test/node-api` (where the test now resides) instead?

Reply:
Yes, it should. Thanks. Fixed.